torch/csrc/jit/ir/alias_analysis.cpp

```diff
@@ -1452,6 +1453,10 @@ class AliasDb::WorkingSet {
 
   const AliasDb& aliasDb_;
   std::vector<Node*> nodes_;
+  // Extra data structure for nodes for faster look up
```

Comment:
Can you describe what the Node is being mapped to (what it represents) instead of how it is being used ?